HarfangLab/harfanglab/ingest/parser.yml

```
@@ -82,6 +82,9 @@ stages:
       - set:
           destination.port: "{{json_event.message.dport}}"
 
+      - set:
+          network.direction: "{{json_event.message.direction}}"
```

According to the ECS specification, the values for `network.direction` are predefined (cf [ECS Specification](https://www.elastic.co/guide/en/ecs/current/ecs-network.html#field-network-direction)).
I suggest the following change:
```suggestion
      - translate:
        dictionary:
          'in': 'inbound'
          'out': 'outbond'
        mapping:
          json_event.message.direction: network.direction
```